routes/api.php

```diff
@@ -17,3 +19,8 @@
 
 Route::get('users/{user}', [UserController::class, 'get'])->name('users.get');
 Route::apiResource('users', UserController::class)->only(['store', 'update']);
+
+Route::group(['middleware' => ['api']], function (Router $router) {
+    $router->get('/invite-key/{inviteKeyId}', [UserController::class, 'getInviteKey']);
```

Comment:
Ik krijg een lege pagina bij http://127.0.0.1:8000/api/invite-key/2 en dat met /1. De rest van de requests lijkt goed te werken.

Hier is mijn database content van invite keys:
![image](https://user-images.githubusercontent.com/10811551/113448151-92914380-93fb-11eb-822a-12e94c11ea3e.png)

Reply:
Dat komt omdat de `InviteKey` niet met surrogaatsleutels werkt. Je kan dus niet zoeken op nummers (wat ook niet heel veilig zou zijn), maar je moet op de `value` zoeken van een `InviteKey`. Wanneer je `api/invite-key/156M` zou zoeken, dan krijg je wel de `InviteKey` terug. 

Dit request wordt dus gebruikt las je een spel wilt joinen en dan type je daar de code in, ook wel de `value`. Hierdoor stuurt de api alleen resultaat terug, als de `InviteKey` ook bestaat. Hieruit kan dan weer een `game_id` worden gehaald.

Reply:
Ik snap het. Ik ben nu alle keys afgegaan van http://127.0.0.1:8000/api/invite-key/156m t/m http://127.0.0.1:8000/api/invite-key/ye3n en bij ongeveer de helft van de keys krijg ik een lege pagina en de andere helft json.

Reply:
Dat had er inderdaad te maken met de opmerking die Leon ook eerder had gemaakt. Er kwam helemaal geen bericht terug uit de api als de code in gebruik is of als je een foutieve code opgeeft. 

Dit heb ik nu dus verandert. Er wordt een foutmelding teruggeven die verstaanbaar is voor de gebruiker.